docs/manual/java/guide/cluster/code/docs/home/persistence/BlogEvent.java

```diff
@@ -16,7 +16,7 @@
 // #sharded-tags
 interface BlogEvent extends Jsonable, AggregateEvent<BlogEvent> {
 
-  int NUM_SHARDS = 20;
+  int NUM_SHARDS = 10;
```

Comment:
Only 9 😞 For 10 not work, as described on https://github.com/lagom/lagom/issues/1677
`LIKE 'Shard1%'` return events for `Shard1` and `Shard10` 😞

Reply:
It's correct to have the number set to 10. It will produce tags from 0 to 9. 

We should be more explicit about this. I will adapt the message.